src/test/ui/enum/enum-discrim-autosizing.stderr

```diff
@@ -1,10 +1,11 @@
-error[E0081]: discriminant value `0` already exists
-  --> $DIR/enum-discrim-autosizing.rs:8:12
+error[E0081]: discriminant value `0` assigned more than once
+  --> $DIR/enum-discrim-autosizing.rs:9:12
    |
 LL |     Au64 = 0,
-   |            - first use of `0`
+   |            - first assignment of `0`
+LL |
 LL |     Bu64 = 0x8000_0000_0000_0000
-   |            ^^^^^^^^^^^^^^^^^^^^^ enum already has `0` (overflowed from `9223372036854775808`)
+   |            ^^^^^^^^^^^^^^^^^^^^^
```

Comment:
huh, actually now that the label is removed, maybe we should point out if it's overloaded.

Reply:
Yeah, that was actually why i wasn't so eager to remove the second label. It was useful when the duplicate also was an overflowing literal. So yeah i agree, at the very least point out if there was an overflow. I'll look into this tomorrow

Reply:
Ok how about this, given two enums:
```
#[repr(u8)]
enum MyEnum1 {
    First = 257,
    Second = 256,
    Last = 513,                            
}                        
                   
#[repr(i8)]           
enum MyEnum2 {                                                            
    First = -1,       
    Second = -2,
    Last,
} 
```
We now error as such:
```
error[E0081]: discriminant value `1` assigned more than once
 --> tst.rs:4:1
  |
4 | / enum MyEnum1 {
5 | |     First = 257,
  | |             --- first assignment of `1` (overflowed from `257`)
6 | |     Second = 256,
7 | |     Last = 513,
  | |            --- second assignment of `1` (overflowed from `513`)
8 | | }
  | |_^

error[E0081]: discriminant value `-1` assigned more than once
  --> tst.rs:11:1
   |
11 | / enum MyEnum2 {
12 | |     First = -1,
   | |             -- first assignment of `-1`
13 | |     Second = -2,
14 | |     Last,
   | |     ---- second assignment of `-1`
15 | | }
   | |_^
   |
note: assigned discriminant for `Last` was incremented from the previous discriminant
  --> tst.rs:13:5
   |
13 |     Second = -2,
   |     ^^^^^^^^^^^

error: aborting due to 2 previous errors
```
I think adding control flow so that we only print the second label if the second assignent is overflowed is redundant, might as well just have the label regardless. I've removed the span of the base-error to the enum itself so that we don't double up on the duplicate variant.

Oh and thanks a lot for sticking with me so far, all of this is quite new to me still so i appreciate the patience.

Reply:
I like this a lot.